includes/ft_printf_list.sh

```diff
@@ -34,7 +34,7 @@ then
 	CHK_FT_PRINTF_LIST2+=("s" "% Z |test")
 	CHK_FT_PRINTF_LIST2+=("s" "% Z%s|test")
 	CHK_FT_PRINTF_LIST+=("s" "%%|test")
-	CHK_FT_PRINTF_LIST2+=("s" "%   %|test")
+	CHK_FT_PRINTF_LIST+=("s" "%   %|test")
```

Comment:
This one is a "bad" use of printf: One space is enough, two or more spaces doesn't mean anything.